pkg/authn/oidc.go

```diff
@@ -38,13 +38,18 @@ var (
 
 // NewOIDCAuthenticator returns OIDC authenticator
 func NewOIDCAuthenticator(ctx context.Context, config *OIDCConfig) (*OIDCAuthenticator, error) {
-	var dynamicCA *dynamiccertificates.DynamicFileCAContent
+	var (
+		dynamicCA         *dynamiccertificates.DynamicFileCAContent
```

Comment:
This is about typed nils, right? Please, add a comment, these are always counterintuitive :neutral_face: